packages/stk/stk_mesh/stk_mesh/base/BulkData.hpp

```diff
@@ -571,9 +571,10 @@ class BulkData {
                          Entity e_to,
                          const RelationIdentifier local_id );
 
-
+#ifndef STK_HIDE_DEPRECATED_CODE // Delete after Oct 2024
```

Comment:
Seems to be a very short notice for this deprecation since we are already in October.